src/panels/lovelace/elements/hui-image-element.js

```diff
@@ -0,0 +1,58 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../components/hui-image.js';
+
+import ElementClickMixin from '../mixins/element-click-mixin.js';
+
+/*
+ * @appliesMixin ElementClickMixin
+ */
+class HuiImageElement extends ElementClickMixin(PolymerElement) {
+  static get template() {
+    return html`
+      <style>
+        :host(.clickable) {
+          cursor: pointer; 
+        } 
+        hui-image {
+          overflow-y: hidden;
+        } 
+      </style>
+      <hui-image
+        hass="[[hass]]"
+        entity="[[_config.entity]]"
+        image="[[_config.image]]"
+        state-image="[[_config.state_image]]"
+        camera-image="[[_config.camera_image]]"
+        filter="[[_config.filter]]"
+        state-filter="[[_config.state_filter]]"
+        title$="[[computeTooltip(hass, _config)]]"
+      ></hui-image>
+    `;
+  }
+
+  static get properties() {
+    return {
+      hass: Object,
+      _config: Object
+    };
+  }
+
+  ready() {
+    super.ready();
+    if (this._config.tap_action !== 'none') {
```

Comment:
This is not a good time to make this check as the config can change via `setConfig`. That's why we check in cards for `this.$` to see if DOM has been created or not.

Reply:
It's different with these elements though right? They can't be the target of a filter card so their content doesn't change, they are either rendered or not.

Reply:
I don't think that we should make that assumption. If we do, add a comment that you know it's a bug but it's not something that is impacting us right now.

Reply:
I'd rather just change it now in that case. I thought we could safely make that assumption for elements

Reply:
Each web component is an enclosure of it's own rendering and business logic and should be able to work in any order its public API is used (`setConfig(config)` and `.hass = hass`)